src/views/templates/list/components/VirtualMachineTemplatesRow.tsx

```diff
@@ -10,12 +12,27 @@ import useWorkloadProfile from '../hooks/useWorkloadProfile';
 import VirtualMachineTemplatesSource from './VirtualMachineTemplatesSource';
 
 const VirtualMachineTemplatesRow: React.FC<
-  RowProps<V1Template, { kind: string; availableTemplatesUID: Set<string> }>
-> = ({ obj, activeColumnIDs, rowData: { kind, availableTemplatesUID } }) => {
+  RowProps<V1Template, { availableTemplatesUID: Set<string> }>
+> = ({ obj, activeColumnIDs, rowData: { availableTemplatesUID } }) => {
+  const history = useHistory();
+
   return (
     <>
       <TableData id="name" activeColumnIDs={activeColumnIDs}>
-        <ResourceLink kind={kind} name={obj.metadata.name} namespace={obj.metadata.namespace} />
```

Comment:
why not using the `onClick` prop to push the url?

Reply:
Ah great!